automation_tools/__init__.py

```diff
@@ -1622,6 +1623,29 @@ def setenforce(mode):
     run('setenforce {0}'.format(mode))
 
 
+def host_pings(host, attempts=100):
+    """This ensures the given IP/hostname pings succesfully.
+
+    :param host: A string. The IP or hostname of host.
+    :param attempts: An integer.
+        The maximum number of attempts to ping the host.
+
+    """
+    for attempt in range(attempts):
```

Comment:
With this number of attempts we are waiting for 8.333333333 minutes.

I think a better approach here is to get the time stamp and check if 20 minutes have passed since the start of the pooling. then we can expect a timeout in minutes to be passed and defaulting to 20.